_overviews/reflection/overview.md

```diff
@@ -171,7 +171,7 @@ which makes all classes and types available that are loaded by the classloader
 that also loaded the class of `p` (`Purchase`), which we need in order to
 access member `shipped`.
 
-    scala> val shippingTermSymb = ru.typeOf[Purchase].decls(ru.TermName("shipped")).asTerm
+    scala> val shippingTermSymb = ru.typeOf[Purchase].declaration(ru.TermName("shipped")).asTerm
```

Comment:
I think that `declaration` is [deprecated in favor of `decl`](https://github.com/scala/scala/blob/v2.12.4/src/reflect/scala/reflect/api/Types.scala#L97-L99) these days...